### DISCUSSION TITLE
Icon for About

### DISCUSSION THREAD

**Discussion by KaigaMunene** [tailwindlabs/heroicons#939] [Ideas]
How comes no one ever created an about icon and most websites have an about section in there nav bar ?

**Comment by zefir-git** [comment]
What is an "about" icon? I have never seen one. Maybe you could use information-circle, user or user-circle (if "about" a person). For a company about maybe building-office, building-office-2, building-storefront (for a shop). Or maybe book-open if it's more like a knowledge base or something.

I genuinely can't picture what icon you‘re suggesting. Googling "about icon" gives me images of icons close to `information-circle`.
